configs/default/LDAR-LDARCF411.config

```diff
@@ -20,63 +22,96 @@ resource LED 1 C13
 resource LED 2 C14
 resource SPI_SCK 1 A05
 resource SPI_SCK 2 B13
-resource SPI_SCK 3 B03
 resource SPI_MISO 1 A06
 resource SPI_MISO 2 B14
-resource SPI_MISO 3 B04
 resource SPI_MOSI 1 A07
 resource SPI_MOSI 2 B15
-resource SPI_MOSI 3 B05
 resource ADC_BATT 1 B00
 resource ADC_CURR 1 B01
-resource FLASH_CS 1 A15
 resource OSD_CS 1 B12
 resource GYRO_EXTI 1 A01
+resource GYRO_EXTI 2 NONE
 resource GYRO_CS 1 A04
 resource USB_DETECT 1 C15
 
-# timer
+# timer list
 timer A03 AF3
 # pin A03: TIM9 CH2 (AF3)
-timer A00 AF1
-# pin A00: TIM2 CH1 (AF1)
-timer B10 AF1
-# pin B10: TIM2 CH3 (AF1)
+timer B04 AF2
+# pin B04: TIM3 CH1 (AF2)
+timer B05 AF2
+# pin B05: TIM3 CH2 (AF2)
 timer B06 AF2
 # pin B06: TIM4 CH1 (AF2)
 timer B07 AF2
 # pin B07: TIM4 CH2 (AF2)
+timer B03 AF1
+# pin B03: TIM2 CH2 (AF1)
+timer B10 AF1
+# pin B10: TIM2 CH3 (AF1)
+timer A00 AF2
+# pin A00: TIM5 CH1 (AF2)
+timer A02 AF2
+# pin A02: TIM5 CH3 (AF2)
 timer A08 AF1
 # pin A08: TIM1 CH1 (AF1)
 
 # dma
 dma ADC 1 0
 # ADC 1: DMA2 Stream 0 Channel 0
-dma pin A00 0
-# pin A00: DMA1 Stream 5 Channel 3
-dma pin B10 0
-# pin B10: DMA1 Stream 1 Channel 3
+dma pin B04 0
+# pin B04: DMA1 Stream 4 Channel 5
+dma pin B05 0
+# pin B05: DMA1 Stream 5 Channel 5
 dma pin B06 0
 # pin B06: DMA1 Stream 0 Channel 2
 dma pin B07 0
 # pin B07: DMA1 Stream 3 Channel 2
+dma pin B03 0
+# pin B03: DMA1 Stream 6 Channel 3
+dma pin B10 0
+# pin B10: DMA1 Stream 1 Channel 3
+dma pin A00 0
+# pin A00: DMA1 Stream 2 Channel 6
+dma pin A02 0
+# pin A02: DMA1 Stream 0 Channel 6
 dma pin A08 0
 # pin A08: DMA2 Stream 6 Channel 0
 
 # feature
+feature -RX_PARALLEL_PWM
+feature RX_SERIAL
+feature SOFTSERIAL
+feature TELEMETRY
 feature OSD
+feature AIRMODE
+feature ANTI_GRAVITY
+feature DYNAMIC_FILTER
+
+# serial
+serial 0 64 115200 57600 0 115200
 
 # master
+set serialrx_provider = SBUS
+set dshot_burst = AUTO
+set dshot_bitbang = OFF
+set motor_pwm_protocol = DSHOT600
```

Comment:
If this setting is used as a default, and if your board is used in combination with older ESCs that do not support Dshot, this will result in the motors spinning up as soon as a battery is connected, potentially resulting in damage or injury.  
Do you assume all responsibility for personal injury and damage resulting from using this default setting?

Reply:
In this situation, the manufacturer has understood and made it clear that it is willing to pay full responsibility.